powerquery-docs/Connectors/DatabricksCloud.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Databricks connector
+description: Provides basic information and prerequisites for using Power Query's Databricks connectors.
+author: George Chow
+
+ms.topic: conceptual
+ms.date: 6/29/2022
+ms.author: George Chow
+LocalizationGroup: reference
+---
+
+# Databricks Cloud
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | Azure Databricks: General Availability<br/>Databricks: Preview |
+| Products | Power BI (Datasets)<br/>Power BI (Dataflows) |
+| Authentication Types Supported | Azure Active Directory<br/>Organizational Account<br/>Personal Access Token |
+| | |
+
+>[!Note]
+>Some capabilities may be present in one product but not others due to deployment schedules and host-specific capabilities.
+
+
+## Prerequisites
+
+You requires either an Azure Databricks account or a paid Databricks Cloud account to use this connector. Databricks Community Edition is not supported.
```

Comment:
I am not sure if "Databricks Cloud account" is commonly used. Is this a commonly used phrase?

if not I would use something like, Databricks on AWS or Databricks on GCP or even we can say Databricks on other clouds.

Reply:
Now that I think about it, the connector is useable with Databricks SQL Warehouse (the current name as far as I can tell: https://docs.databricks.com/sql/admin/sql-endpoints.html) which is only available in paid form. I want to communicate that the connector does not work for Databricks Community Edition. 

Amended.